doc/aurpublish.1.asciidoc

```diff
@@ -0,0 +1,83 @@
+aurpublish(1)
+=============
+
+Name
+----
+aurpublish - PKGBUILD management framework for the Arch User Repository
+
+
+Synopsis
+--------
+'aurpublish' [options] PACKAGE
+'aurpublish' log [<options>] PACKAGE
+'aurpublish' setup
+
+
+Description
+-----------
+Aurpublish is an utility which helps maintainers of aur packages.
+
+
+How it works
+------------
+Commit PKGBUILDs in named subdirectories.
+Export them to the AUR with the aurpublish command, using the subtree push 
+stratagem. This preserves an independent history for third-party hosting, 
+pull requests... ;)
+
+Commands
+--------
+
+setup::
+        Initialize a new repository with git hooks.
+
+log <PACKAGE>::
+        View the git log of a package subtree.
+
+Options
+-------
+
+-p <PACKAGE>::
+        Instead of publishing, pull changes from the AUR.
+        Can import packages into a new subtree.
+
+-s, --speedup <PACKAGE>::
+        Speedup future publishing by recording the subtree history during 
+        a push. This creates a merge commit and a second copy of all 
+        commits in the subtree. For more details, see the "--rejoin" 
+        option in git subtree.
```

Comment:
We can use the linkman macro from pacman to turn git-subtree into a manpage link... hmm.

Reply:
Sorry, but I'm not sure if I understood what you mean at all :)

Reply:
https://git.archlinux.org/pacman.git/tree/doc/asciidoc.conf

adding a2x --asciidoc-opts "-f doc/asciidoc.conf" will let you use macros defined in this attached file.

`linkman:git-subtree[1]` then turns into a nicely formatted reference.

See e.g. https://git.archlinux.org/arch-install-scripts.git/commit/?id=aeab95dbab2bac426ef8ab45056b83e2e6a888a0 for how I added the same to arch-install-scripts.

...

I could add this later, I suppose.

Reply:
See c126c24b92ea8e533c1953565ed978d2a9d14e24

(needs rebasing)